charts/kube-state-metrics/Chart.yaml

```diff
@@ -7,7 +7,7 @@ keywords:
 - prometheus
 - kubernetes
 type: application
-version: 3.1.0
+version: 3.2.0
```

Comment:
Can you rebase and bump this to 3.3.0? Thanks!

Reply:
Sure, its ready to go now